src/cljs/ataru/virkailija/editor/view.cljs

```diff
@@ -45,34 +44,34 @@
    "Uusi lomake"])
 
 (defn- copy-form []
```

Comment:
The outer function is a "component-creation function"  and the inner function is "the render function". Now the values never change because the render-function (the (fn []...)) always sees the same value. The old implementation sees the changes in values. [See Form-2 functions in re-frame docs.](https://github.com/Day8/re-frame/wiki/Creating-Reagent-Components#form-2--a-function-returning-a-function)